### DISCUSSION TITLE
Add auto files (.md) generartion for all project modules

### DISCUSSION THREAD

**Discussion by zekriHichem** [mkdocs/mkdocs#2903] [Q&A]
Hi, 

I think if there is a plugin that allows to generate automatically all the project tree like a mkdocs project we put in each file the docstring of the concerned module. 

Example in sphinx : `python sphinx-apidoc -T -M --separate -o docs src/app/`

**Comment by pawamoy** [comment]
Hi, is this a question? MkDocs does not natively offer auto-documention from code. Take a look at the [mkdocstrings](https://GitHub.com/mkdocstrings/mkdocstrings) plugin for that.
